SubscribeToMailchimp.module

```diff
@@ -32,11 +32,19 @@ class SubscribeToMailchimp extends WireData implements Module, ConfigurableModul
         if(isset($email)) {
             $api = $this->getApiBase($list, 'members');
             $param['email_address'] = $email;
+
             foreach ($merge_fields as $k => $v) {
                 if(!empty(trim($v))) { // avoid to override data of existing leads with empty field values
```

Comment:
Thanks a lot! Trim produces a warning, if an array is given. Can you maybe switch these two somehow, so that the trim won't apply if you provide an array in $v?